### PR TITLE
Removing display of weekdays when sat or sun is included with weekdays

### PR DESCRIPTION
**Pull request by hollyegrayson**

### CODE REVIEW

File: src/app/components/AssignmentDefaultRecurrenceDetails.tsx [hunk 1]
```diff
@@ -15,7 +15,11 @@ export interface AssignmentDefaultRecurrenceDetailsProps {
 export default class AssignmentDefaultRecurrenceDetails extends React.PureComponent<AssignmentDefaultRecurrenceDetailsProps, any> {
     render() {
         const { data: { days, startTime, endTime, sheriffsRequired } } = this.props;
-        const dayDisplay = EnumUtils.displayEnum(DaysOfWeek, days).join(", ");
+        
+        let dayDisplay = EnumUtils.displayEnum(DaysOfWeek, days).join(", ");
```
Comment (cjam) [inline src/app/components/AssignmentDefaultRecurrenceDetails.tsx]: Can we add some of these cases to the unit tests for this utility?  Or is it very specific to the days of the week?

Comment (hollyegrayson) [reply src/app/components/AssignmentDefaultRecurrenceDetails.tsx]: I think it is pretty specific to the days of the week

Comment (cjam) [reply src/app/components/AssignmentDefaultRecurrenceDetails.tsx]: hmm, in that case.  We may want to wrap the enumUtils in a function specific for days of the week and then write tests for it.

Comment (hollyegrayson) [reply src/app/components/AssignmentDefaultRecurrenceDetails.tsx]: okay sound good

Comment (cjam) [reply src/app/components/AssignmentDefaultRecurrenceDetails.tsx]: A little trick in typescript, you can make functions look like static members of enums by using namespaces.  i.e.:

```ts
export enum DaysOfWeek {
    Mon = 1 << 0,
    Tue = 1 << 1,
    Wed = 1 << 2,
    Thu = 1 << 3,
    Fri = 1 << 4,
    Sat = 1 << 5,
    Sun = 1 << 6,
    Everyday = Mon | Tue | Wed | Thu | Fri | Sat | Sun,
    Weekdays = Mon | Tue | Wed | Thu | Fri
}

export namespace DaysOfWeek{
    export function display(value:DaysOfWeek):string{
        // Display logic here
    }
}

const test:DaysOfWeek = DaysOfWeek.Sat;
DaysOfWeek.display(test);
```

File: src/app/api/Api.ts [hunk 2]
```diff
@@ -21,6 +22,26 @@ export enum DaysOfWeek {
     Weekdays = Mon | Tue | Wed | Thu | Fri
 }
 
+export namespace DaysOfWeek {
+    
+    export function getDisplayValues(value: DaysOfWeek): string[] {
```
Comment (cjam) [inline src/app/api/Api.ts]: 👌

File: src/app/api/__tests__/ApiTest.ts [hunk 3]
```diff
@@ -21,3 +22,21 @@ describe('API Client',()=>{
 
 })
 
+describe('DaysOfWeek.getDisplayValues', () => {
+    const util = DaysOfWeek.getDisplayValues;
+    
+    it('Should return each day if weekdays and Sat or Sun is selected', () => {
```
Comment (cjam) [inline src/app/api/__tests__/ApiTest.ts]: perhaps a case for "Weekdays"?

i.e. if ```DaysOfWeek.Weekdays``` or ```DaysOfWeek.Mon|DaysOfWeek.Tues.......|DaysOfWeek.Fri```

**Pull request merged**